src/main/kotlin/routes/Renderer.kt

```diff
@@ -0,0 +1,18 @@
+package routes
+
+import com.mitchellbosecke.pebble.PebbleEngine
+import java.io.StringWriter
+import java.util.*
+import kotlin.collections.HashMap
+
+object Renderer {
```

Comment:
To return HTML pages, you can use the `hexagon_web` module with helpers coded specifically for this regard :) Sorry but this feature is not properly documented yet. However, I'm telling you just in case you want to find out ;)